packages/client-sdk-nodejs/src/cache-client.ts

```diff
@@ -51,6 +51,24 @@ export class CacheClient extends AbstractCacheClient implements ICacheClient {
     this.logger.info('Creating Momento CacheClient');
   }
 
+  /**
+   * Creates a new instance of CacheClient that eagerly creates its connections to Momento. By default, connections are
+   * created lazily when the client is first used. If connections cannot be established, an error will be logged and
+   * execution will resume.
+   * @param {CacheClientProps} props configuration and credentials for creating a CacheClient.
+   * @param {number} timeoutSeconds how long to wait for the connections.
+   */
+  static async createWithEagerConnections(
```

Comment:
maybe just name this singular since we hope that most users don't have to think about how many connections we have? I don't feel too strongly but I think that might fit a little better with our existing config facade.

Reply:
also we should add a plain `create` factory function to correspond to this one. That way we can start to recommend the factory functions for both use cases, rather than having one workflow require a `new` and the other one not.